### ISSUE TITLE
**[Suggestion] Trouver Parchment (l) items

### ISSUE THREAD

**Issue by Mastalas** [dillydill123/inventory-setups#224]
I recently used a Trouver Parchment on some items I use in multiple equipment setups, and I thought "Toggle Fuzzy" would work with this, but it doesn't. Even if you swap out the item in equipment for the new (l) item, it still doesn't register. Ideally, it would be great if "Toggle Fuzzy" worked on trouver'd items. Are trouver'd items available in the API? I'd be forever grateful if this was possible, or my equipment setups will forever be a little more annoying. XD

**Comment by dillydill123** [comment]
So even if you have the exact item in the slot it still doesn't work? I'm surprised fuzzy doesn't automatically work. I don't know if there is a trouver item API, but I get the variation mapping from RuneLite API (then add additional items after). So worst case I can add them manually

**Comment by Mastalas** [comment]
Yeah for whatever reason, it doesn't seem to recognize it as an item when you equipped it and then update the item slot. Of course I haven't trouver'd every item but I know the imbued god capes don't work. There is a decent amount of items to manually add, but here is the list in case you have a chance. Thanks for the quick reponse!

https://oldschool.runescape.wiki/w/(l)#:~:text=(l)%20is%20a%20suffix%20applied,protect%20the%20item%20upon%20death.

**Comment by dillydill123** [comment]
I'm realizing now, it might be the case that I need to add the locked versions of the god capes to the variation mapping here:

https://github.com/dillydill123/inventory-setups/blob/master/src/main/java/inventorysetups/InventorySetupsVariationMapping.java#L132

Are you able to try with some of the other locked items? like a fire cape for example?

**Comment by 1Defence** [comment]
> I'm realizing now, it might be the case that I need to add the locked versions of the god capes to the variation mapping here:
> 
> https://github.com/dillydill123/inventory-setups/blob/master/src/main/java/inventorysetups/InventorySetupsVariationMapping.java#L132
> 
> Are you able to try with some of the other locked items? like a fire cape for example?

After testing, this only occurs with locked items that have been manually added to InventorySetupsVariationMapping. 
Adding the locked variant does not resolve the issue, but instead removing the imbued cape from that list solves it. so theres likely an issue with how that extra variation mapping is handled.

Works, shows locked variants

```
		final int itemIDImbuedGodCape = 1000000002;
		//mappings.put(IMBUED_SARADOMIN_CAPE, itemIDImbuedGodCape);
```

Does not show locked variants

```
		final int itemIDImbuedGodCape = 1000000002;
		mappings.put(IMBUED_SARADOMIN_CAPE, itemIDImbuedGodCape);
```

Additionally does not show locked variants

```
		final int itemIDImbuedGodCape = 1000000002;
		mappings.put(IMBUED_SARADOMIN_CAPE, itemIDImbuedGodCape);
		mappings.put(IMBUED_SARADOMIN_CAPE_L, itemIDImbuedGodCape);
```

Further more, items not on this mapping with locked variants such as the infernal cape work as intended.

**Comment by dillydill123** [comment]
Is this still an issue?

**Comment by dowodenum** [comment]
> Is this still an issue?

I think so. I trouver'd my rune pouch, and even with fuzzy, on click it pops up a context menu showing "Duplicate-item" and "Remove-layout", without actually withdrawing the item.

**Comment by dillydill123** [comment]
Closing, reopen if this is still an issue.
